portal/exercise_diet/templates/exercise_diet/recipe.html

```diff
@@ -1,19 +1,17 @@
-{% block body %}
-    {% trans %}<button type="button" class="btn btn-secondary-top" data-dismiss="modal"><b>close</b><span>X</span></button>{% endtrans %}
-    {{ asset|safe }}
-    <div class="modal-nav">
-        {% if prev_heading != '' %}
-        <span class="left-button">
-           <span> <a href="javascript:show_recipe('{{ prev_heading }}','{{ prev_item_index }}','{{ prev_asset_recipe_type }}');">{{ prev_asset_title|safe }}</a></span>
-        </span>
-        {% endif %}
-        {% if next_heading != '' %}
-        <span class="right-button">
-            <span><a href="javascript:show_recipe('{{ next_heading }}','{{ next_item_index }}','{{ next_asset_recipe_type }}');">{{ next_asset_title|safe }}</a></span>
-        </span>
-        {% endif %}
-    </div>
-    <div class="modal-footer">
-        {% trans %}<button type="button" class="btn btn-secondary" data-dismiss="modal"><b>close</b><span>X</span></button>{% endtrans %}
-    </div>
-{% endblock %}
+{% trans %}<button type="button" class="btn btn-secondary-top" data-dismiss="modal"><b>close</b><span>X</span></button>{% endtrans %}
+{{ asset|safe }}
+<div class="modal-nav">
+    {% if prev_heading != '' %}
+    <span class="left-button">
+       <span> <a href="javascript:show_recipe('{{ prev_heading }}','{{ prev_item_index }}','{{ prev_asset_recipe_type }}');">{{ prev_asset_title|safe }}</a></span>
```

Comment:
This still counts as inline javascript, but could be easily moved to an onClick handler in a separate file, passed the arguments as attributes of an HTML tag.

Reply:
I generally agree, but note that this is all the CHCR code. I think we should save changes like this (rewriting working E&D code to match more closely to our standards) for a larger CHCR code revamp, and just leave this for now, so we can get the changes in for Mo to review.

Reply:
Agreed, just something I want us to be aware of :)